shared_model/builders/protobuf/builder_templates/query_template.hpp

```diff
@@ -93,20 +93,29 @@ namespace shared_model {
 
       auto createdTime(interface::types::TimestampType created_time) const {
         return transform<CreatedTime>([&](auto &qry) {
-          qry.mutable_payload()->set_created_time(created_time);
+          auto *meta = new iroha::protocol::QueryPayloadMeta(
```

Comment:
Please avoid `new` calls. Something similar to following should work:
```C++
qry.mutable_payload()->mutable_meta()->set_created_time(created_time);
```

This should work because `mutable_x` methods look like this:
```C++
inline ::iroha::protocol::Transaction_Payload* Transaction::mutable_payload() {
  
  if (payload_ == NULL) {
    payload_ = new ::iroha::protocol::Transaction_Payload;
  }
  // @@protoc_insertion_point(field_mutable:iroha.protocol.Transaction.payload)
  return payload_;
}
```